### ISSUE TITLE
tornadotoad

### ISSUE THREAD

**Issue by asmodehn** [garethdmm/gryphon#23]
tornadotoad seems to rely on tornado 4.*
with tornado 5 it throws errors about IOLoop (when dashboard errors), so on my venv I rolledback to tornado 4.5.3.

[tornadotoad](https://pypi.org/project/tornadotoad/) doesnt seem very maintained... the github repo is gone...
What would be the plan going forward for this ?

**Comment by garethdmm** [comment]
Agreed if tornadotoad isn't well maintained we should switch to something else soon. That's mostly a planning/research task I think. I'm not familiar with the current alternatives, but I'm sure they must exist. It's possible we could even just switch to something similar to what the strategy engine uses.
